suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -66,9 +71,17 @@ def get_arguments(parser: ArgumentParser):  # pragma: no cover
         args: the parsed args from the parser
     """
     parser.add_argument(
-        "--access_key",
-        env_var="ACCESS_KEY",
-        help="the access key for ACRCloud (required)",
+        "--bearer-token",
```

Comment:
I think we should either change all arguments to dashes or make the new ones use underscores. I would tend to dashes nowadays.

Reply:
I'm a dashes person myself, it might be easier to change the reset in a separate pr so it doesn't impact landing the excel and other changes. wdyt?

Reply:
Yes, let's do it in a later cleanup.